### DISCUSSION TITLE
Short film made entirely with ToonSquid!

### DISCUSSION THREAD

**Discussion by ivana67dj** [keiwando/toonsquid#235] [Show and tell]
I started using ToonSquid a few months ago and I decided to take a chance with it for my Bachelor's thesis :) My first animation ever. It engages with topics of gender and folklore and how I relate to all that. 

I hope you enjoy!

https://youtu.be/EukScg5d9AA

**Comment by keiwando** [comment]
Wow, that's great! Thank you so much for sharing this video. 
I am honoured that you decided to work with ToonSquid as part of your Bachelor's thesis!

**Comment by MojoMan49** [comment]
Ivana — Perhaps the finest use of ToonSquid I’ve see to date! I threw this together a few days ago and it doesn’t even have  a audio track

https://user-images.githubusercontent.com/127110520/231267506-51c70b20-6631-4021-ab7e-ec04908a9d63.MOV

**Comment by PixelNoire** [comment]
Nice film! Did you use multiples scenes to stay organized?

**Reply by MojoMan49** [reply]
Thanks for the positive feedback.

I was pretty pro-scene in the beginning and can see the utility down the road, but in this instance it was all about each character being  layer and then using the “keyframe” tools to move, scale fade, etc.

To be honest I don’t know if this is the most efficient use of keyframe since lots of samples movies I see so more cuts to the layer timeline.

I also tend to do slower fps if I see this as a potential gif in the end.
![6CB98F1A-9C45-48DF-995B-64D2947CD4DA](https://user-images.githubusercontent.com/127110520/232138468-a76f4b71-2105-459b-9c6f-bca9d7c83904.png)

If you have any other Qs I can help you with or at least clarify, please feel free

**Comment by PixelNoire** [comment]
Thanks!
